examples/generative/corrdiff/train.py

```diff
@@ -260,10 +271,20 @@ def main(cfg: DictConfig) -> None:
         with open(os.path.join(c.run_dir, "training_options.json"), "wt") as f:
             json.dump(c, f, indent=2)
 
+    # create a training dataset
     (dataset, dataset_iterator) = init_dataset_from_config(
         dataset_cfg, data_loader_kwargs, batch_size=batch_size_gpu, seed=seed
     )
-
+    
+    # create a valiadtion dataset
+    validation_dataset_cfg = copy.deepcopy(dataset_cfg)
+    validation_dataset_cfg['all_times'] = False
+    validation_dataset_cfg['train'] = False
```

Comment:
This will only work with dataloaders that have the parameters `all_times` and `train` (CWB will work but any future dataloaders might be configured differently).

Reply:
@jleinonen I have modified the use of this call by moving the sprint to a wrapper function inside dataset, where in train.py the default is false and the behavior should ignore the validation test split if none is provided in the yaml